File: Challenges/Coding World Cup/2023/Day 19 - PAK vs AFG - Best Lineup.cpp
long long int bestLineup(int n, vector<int> x)
{
    // Write your code here.
    sort(x.begin(), x.end());

    long long ans = 0;

    for (int i = 0; i < n; i++)
    {
        ans += (x[n - 1 - i] - i) * (x[n - 1 - i] - i);
    }

    return ans;
}

File: Challenges/Coding World Cup/2023/Day 19 - PAK vs AFG - Best Lineup.java
import java.util.Arrays;

public class Solution {
    static long bestLineup(int n, int[] x) {
        Arrays.sort(x);

        long ans = 0;

        for (int i = 0; i < n; i++) {
            ans += (long) (x[n - 1 - i] - i) * (x[n - 1 - i] - i);
        }

        return ans;
    }
}